bridgetown-core/lib/bridgetown-core/configurations/netlify/netlify.sh

```diff
@@ -3,12 +3,11 @@
 
 echo "Updating netlify.toml with references to our built files"
 
-CSS_PATH=`find output/_bridgetown/static/css/*.css -type f | sed -e 's,output\/,/,g'`
-JS_PATH=`find output/_bridgetown/static/js/*.js -type f | sed -e 's,output\/,/,g'`
+CSS_PATH=`find output/_bridgetown/static/*.css -type f | sed -e 's,output\/,/,g'`
```

Comment:
@jaredcwhite is this path universal irrespective of front-end bundling option? Or is it specific to esbuild only?